docs/api-client/README.md

```diff
@@ -24,8 +24,8 @@ GET /template/raw/:title
 
```

Comment:
we can set up to run `npm run format` via CI as per discussion with @mroth . i'll make an issue for that